training/src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,13 @@
+package wolox.training.repositories;
+
+import java.util.List;
+import org.springframework.data.repository.Repository;
+import org.springframework.stereotype.Component;
+import wolox.training.models.Book;
+
+@Component
+public interface BookRepository extends Repository<Book, Long> {
+
+    public Book findFirstByAuthor(String author);
```

Comment:
With my previous comment, the signature of this method would be: 
public Optional< Book > findFirstByAuthor(String author);